app/serializers/user.js

```diff
@@ -5,9 +5,13 @@ export default ApplicationSerializer.extend({
   serialize(snapshot, options) {
     const json = this._super(...arguments);
     if (snapshot.id) {
-      let attributesToOmit = ['email'];
+      let attributesToOmit = [];
       if (!snapshot.adapterOptions || !snapshot.adapterOptions.includePassword) {
         attributesToOmit.push('password');
+      } else if (!snapshot.adapterOptions || !snapshot.adapterOptions.includeEmail) {
```

Comment:
@niranjan94 I saw that. I implemented like that only. What is happening is by the following code:
```
import ApplicationSerializer from 'open-event-frontend/serializers/application';
import { pick, omit } from 'lodash';

export default ApplicationSerializer.extend({
  serialize(snapshot, options) {
    const json = this._super(...arguments);
    if (snapshot.id) {
      let attributesToOmit = [];
      if (!snapshot.adapterOptions || !snapshot.adapterOptions.includePassword) {
        attributesToOmit.push('password');
      }
      json.data.attributes = omit(json.data.attributes, attributesToOmit);
      console.log(json.data.attributes);
    } else if (options && options.includeId) {
      json.data.attributes = pick(json.data.attributes, ['email', 'password']);
    }
    return json;
  }
});
```
password is getting ommitted. Still console shows me the error that 'Missing field required' for password.
![email-api8](https://user-images.githubusercontent.com/17925669/28498043-e9d57448-6fb2-11e7-9557-dfa93870d41e.png)